From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Samuel Attard <marshallofsound@electronjs.org>
Date: Mon, 6 Jul 2020 16:59:17 -0700
Subject: remove some deps that do not work on arm64

Once these deps have been fixed upstream we can remove this patch.

diff --git a/.vpython b/.vpython
index e76e966921da472fdf713eabe25235fb5951d0fc..031725b7b0417e456df5dab514382beb3e720226 100644
--- a/.vpython
+++ b/.vpython
@@ -53,10 +53,10 @@ wheel: <
 #
 # This version must be compatible with the version range specified by
 # //third_party/catapult/telemetry/telemetry/internal/util/external_modules.py
-wheel: <
-  name: "infra/python/wheels/psutil/${vpython_platform}"
-  version: "version:5.2.2"
->
+#wheel: <
+#  name: "infra/python/wheels/psutil/${vpython_platform}"
+#  version: "version:5.2.2"
+#>
 
 # Used by:
 #   build/lacros/test_runner.py
@@ -74,37 +74,37 @@ wheel: <
 # This version must also be compatible with the "numpy" version, as "cv2"
 # depends on "numpy". In this case, OpenCV@2.4.13.2 has been built against
 # numpy@1.11.3.
-wheel: <
-  name: "infra/python/wheels/opencv_python/${vpython_platform}"
-  version: "version:2.4.13.2"
-  match_tag: <
-    platform: "win32"
-  >
-  match_tag: <
-    platform: "win_amd64"
-  >
-  match_tag: <
-    abi: "cp27mu"
-    platform: "manylinux1_i686"
-  >
-  match_tag: <
-    abi: "cp27mu"
-    platform: "manylinux1_x86_64"
-  >
-  match_tag: <
-    platform: "macosx_10_6_intel"
-  >
->
+#wheel: <
+#  name: "infra/python/wheels/opencv_python/${vpython_platform}"
+#  version: "version:2.4.13.2"
+#  match_tag: <
+#    platform: "win32"
+#  >
+#  match_tag: <
+#    platform: "win_amd64"
+#  >
+#  match_tag: <
+#    abi: "cp27mu"
+#    platform: "manylinux1_i686"
+#  >
+#  match_tag: <
+#    abi: "cp27mu"
+#    platform: "manylinux1_x86_64"
+#  >
+#  match_tag: <
+#    platform: "macosx_10_6_intel"
+#  >
+#>
 
 # Used by:
 #   third_party/catapult
 #
 # This version must be compatible with the version range specified by
 # //third_party/catapult/telemetry/telemetry/internal/util/external_modules.py
-wheel: <
-  name: "infra/python/wheels/numpy/${vpython_platform}"
-  version: "version:1.11.3"
->
+#wheel: <
+#  name: "infra/python/wheels/numpy/${vpython_platform}"
+#  version: "version:1.11.3"
+#>
 
 # Used by:
 #   third_party/catapult
@@ -139,10 +139,10 @@ wheel: <
 
 # Used by:
 #   testing/buildbot/generate_buildbot_json_coveragetest.py
-wheel: <
-  name: "infra/python/wheels/coverage/${vpython_platform}"
-  version: "version:4.3.4"
->
+#wheel: <
+#  name: "infra/python/wheels/coverage/${vpython_platform}"
+#  version: "version:4.3.4"
+#>
 
 # Used by:
 #   build/android/pylib/local/emulator/avd.py
@@ -173,27 +173,27 @@ wheel: <
 
 # Used by:
 #   tools/perf/flakiness_cli
-wheel: <
-  name: "infra/python/wheels/pandas/${vpython_platform}"
-  version: "version:0.23.4"
-  match_tag: <
-    platform: "win32"
-  >
-  match_tag: <
-    platform: "win_amd64"
-  >
-  match_tag: <
-    abi: "cp27mu"
-    platform: "manylinux1_i686"
-  >
-  match_tag: <
-    abi: "cp27mu"
-    platform: "manylinux1_x86_64"
-  >
-  match_tag: <
-    platform: "macosx_10_6_intel"
-  >
->
+#wheel: <
+#  name: "infra/python/wheels/pandas/${vpython_platform}"
+#  version: "version:0.23.4"
+#  match_tag: <
+#    platform: "win32"
+#  >
+#  match_tag: <
+#    platform: "win_amd64"
+#  >
+#  match_tag: <
+#    abi: "cp27mu"
+#    platform: "manylinux1_i686"
+#  >
+#  match_tag: <
+#    abi: "cp27mu"
+#    platform: "manylinux1_x86_64"
+#  >
+#  match_tag: <
+#    platform: "macosx_10_6_intel"
+#  >
+#>
 wheel: <
   name: "infra/python/wheels/pytz-py2_py3"
   version: "version:2018.4"
